modules/renter.go

```diff
@@ -52,6 +52,7 @@ type FileInfo struct {
 	Filesize       uint64            `json:"filesize"`
 	Available      bool              `json:"available"`
 	Renewing       bool              `json:"renewing"`
+	Redundancy     float64           `json:"redundancy"`
 	UploadProgress float64           `json:"uploadprogress"`
```

Comment:
oh, that's a good point. It'll say Upload 0 until the upload loop has gotten to every single chunk, at which point you are hopefully already at full availability and then some. So you jump from 0 to like 1.5 or something. Which is not acceptable UX. So I think it makes a lot of sense to have both in that context.